frb_codegen/src/generator/dart/ty.rs

```diff
@@ -56,50 +57,64 @@ impl<'a> TypeDartGenerator<'a> {
                 ir,
                 context,
                 dart_api_class_name,
+                is_exception: false,
             }
             .into(),
             Delegate(ir) => TypeDelegateGenerator {
                 ir,
                 context,
                 dart_api_class_name,
+                is_exception: false,
             }
             .into(),
             PrimitiveList(ir) => TypePrimitiveListGenerator {
                 ir,
                 context,
                 dart_api_class_name,
+                is_exception: false,
             }
             .into(),
             Optional(ir) => TypeOptionalGenerator {
                 ir,
                 context,
                 dart_api_class_name,
+                is_exception: false,
             }
             .into(),
             GeneralList(ir) => TypeGeneralListGenerator {
                 ir,
                 context,
                 dart_api_class_name,
+                is_exception: false,
             }
             .into(),
-            StructRef(ir) => TypeStructRefGenerator {
-                ir,
-                context,
-                dart_api_class_name,
+            StructRef(ir) => {
+                let is_exception = ir.is_exception;
+                TypeStructRefGenerator {
+                    ir,
+                    context,
+                    dart_api_class_name,
+                    is_exception,
+                }
+                .into()
             }
-            .into(),
             Boxed(ir) => TypeBoxedGenerator {
                 ir,
                 context,
                 dart_api_class_name,
+                is_exception: false,
             }
             .into(),
-            EnumRef(ir) => TypeEnumRefGenerator {
-                ir,
-                context,
-                dart_api_class_name,
+            EnumRef(ir) => {
+                let is_exception = ir.is_exception;
```

Comment:
Seems that `self.is_exception` === `ir.is_exception`? If so, can we remove the is_exception field from dart generator struct - since the info is already in ir